src/test/java/com/solvd/laba/BaseTest.java

```diff
@@ -14,6 +18,11 @@ public abstract class BaseTest {
     protected HomePage homePage;
     protected final Logger logger = LogManager.getLogger(getClass());
 
+    protected TestDataConfig testData = new TestDataConfig();
+
+    protected String email = testData.getProperty("email");
+    protected String password = testData.getProperty("password");
+
     @BeforeClass
     public void setUp() {
```

Comment:
if you're running tests in parallel, this could cause issues due to shared state